extension/test/newman/globals.json

```diff
@@ -2,6 +2,12 @@
   "id": "0bc0e5bb-608c-4377-af65-815c11754075",
   "name": "Magento Connect User",
   "values": [
+    {
```

Comment:
I think we can remove this, username and pass from globals since we have the auto-generating environment file